docs/website/docs/reference/tuning.md

```diff
@@ -23,12 +23,14 @@ User-provided tuning specs are a mechanism that allows for users to get the
 best performance on custom models and hardware targets without having to modify
 the compiler source code or needlessly special-case compiler heuristics.
 
-Currently, the dispatch tuner that generates tuning specs is still experimental
-and hosted
-[in an external repo](https://github.com/nod-ai/shark-ai/tree/main/tuner). This
-document describes how to work with tuning specs generated by the SHARK Tuner
-or produced manually, but it does not go into detail on how to generate these
-specs.
+This document describes how to work with tuning specs generated by the SHARK
+Tuner or produced manually, but it does not go into detail on how to generate
+these specs. Currently, the dispatch tuner that generates tuning specs is still
+experimental and hosted
```

Comment:
```suggestion
under development and hosted
```
I remember experimental was dropped in some other places thus we maybe shouldn't call it that way here I guess.

Reply:
+1, don't call it experimental at this point. Could still mention status ("under development", "under active development", "evolving"), though such language tends to get outdated when written into docs :P